src/components/DelimitedList/DelimitedList.js

```diff
@@ -67,16 +67,16 @@ class DelimitedList extends Component {
 }
 
 DelimitedList.propTypes = {
-  /** @type {string} class name for rendered content. */
+  /** Provide an optional class to be applied to the containing node */
```

Comment:
What's the reasoning for removing jsdoc info? 🤔 
Even if storybook doesn't necessarily rely on it, that's still valuable info to have in context + if we ever need it for populating a different documentation site/tool/page.

Reply:
It's redundant given the types are defined by `PropTypes`